README.md

```diff
@@ -4,11 +4,73 @@ This repository contains the source code for SKDB and its associated
 components.
 
 
-## Dependencies
+## Running Prepackaged SKDB
 
-You will need at least the following installed:
+You can run the latest release of SKDB's development server with Docker:
 
- * LLVM 15.
- * wasm-ld 15 (this is part of LLVM, but your distribution might split it into
-   a separate package: your wasm-ld version must match your LLVM version).
- * typescript 5.2 or later.
+```sh
+docker run -it -p 3586:8080 skiplabs/skdb-dev-server
+```
+
+The three SKDB packages (`skdb`, `skdb-dev`, and `skdb-react`) are all
+available as normal NPM packages.
+
+
+## Building From Source
+
+It is recommended that you use the prepackaged version of SKBD in most
+situations.
+
+For those who need to build from source, you will need a number of
+dependencies installed: SKDB's
+[`Dockerfile`](https://github.com/SkipLabs/skdb/blob/main/Dockerfile) contains
+the definitive list for Ubuntu, and will give users of other operating systems
```

Comment:
If we want to switch to e.g. debian this reference to ubuntu will become stale and we won't notice. Maybe:
```suggestion
the definitive list used for SKDB's CI, and will give users of other operating systems
```

Reply:
You're right, but also I don't see how we can change this: the Dockerfile only tells you what's right for Ubuntu. It's not even necessarily right for Debian (though it will probably work) and for other distributions, let alone other OSs, with different packages you've potentially got some guesswork to do, even with the Dockerfile as a partial guide.

Reply:
Yes I agree that it is ubuntu specific, but the dockerfile itself states that, so to me it seems more robust to leave that detail out of the sentence that essentially just points people to the dockerfile. Once they see it, it will be clear that it is for ubuntu, and some adjustment will be needed for other OSes.

Reply:
Fixed in e801d35e.